src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const styleDeclarations = sourceString.split(';');
+
+  styleDeclarations.forEach((style) => {
+    const parts = style.split(':');
```

Comment:
Splitting the style declaration by ':' is a good approach, but consider handling cases where the property or value might include a ':' character, such as in a URL or data URI.